### PR TITLE
fix python api bug

### PR DESCRIPTION
**Pull request by zhenwei-intel**
## Type of Change

bug fix

compute_dtype is overwirted by f0a6fd8b4635aa3020899b0fe302b1fd11534ccd

## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Review by hshen14 — APPROVED** [review]

File: intel_extension_for_transformers/llm/runtime/graph/__init__.py [hunk 1]
```diff
@@ -48,14 +48,14 @@ def __import_package(self, model_name):
             import intel_extension_for_transformers.llm.runtime.graph.opt_cpp as cpp_model
         elif model_name == "bloom":
             import intel_extension_for_transformers.llm.runtime.graph.bloom_cpp as cpp_model
-        elif model_name == "chatglm2":
+        elif model_name == "chatglm":
```
Comment (a32543254) [inline intel_extension_for_transformers/llm/runtime/graph/__init__.py:51]: please sync with zhenzhong, I believe we support chat glm v1 and v2

**Review by a32543254 — APPROVED** [review]

File: intel_extension_for_transformers/llm/runtime/graph/__init__.py (hunk 1 above)
Comment (a32543254) [reply intel_extension_for_transformers/llm/runtime/graph/__init__.py:51]: model_name == "chatglm" or model_name == "chatglm2"

**Comment by a32543254** [comment]
please also consider add python ut or extension test for feature.

**Review by PenghuiCheng — APPROVED** [review]

File: intel_extension_for_transformers/llm/runtime/graph/__init__.py (hunk 1 above)
Comment (zhenwei-intel) [reply intel_extension_for_transformers/llm/runtime/graph/__init__.py:51]: fixed

**Pull request merged**